workers/common/src/main/kotlin/common/OrtRunService.kt

```diff
@@ -33,6 +33,7 @@ import org.eclipse.apoapsis.ortserver.model.NotifierJob
 import org.eclipse.apoapsis.ortserver.model.OrtRun
 import org.eclipse.apoapsis.ortserver.model.ReporterJob
 import org.eclipse.apoapsis.ortserver.model.ScannerJob
+import org.eclipse.apoapsis.ortserver.model.WorkerJob
```

Comment:
Commit message nit: To me this is not really extracting logic in the sense of refactoring, but as a first step just introducing a new function along with tests.

Reply:
I have rewritten the commit message accordingly.